.gitignore

```diff
@@ -15,5 +15,6 @@ node_modules/
 /doc/api
 /pkg/*/doc/api
 
-# Generated protocol buffer files.
+# Generated Dart files.
 *.pb*.dart
+*.g.dart
```

Comment:
This is a little odd, since the async files are also generated but _don't_ have a marked filename. Might it be simpler to just explicitly ignore the `deprecation.dart` file?

Reply:
I was thinking in terms of having something to distinguish generated code that's not checked in, and I'd seen the `.g.dart` pattern with some other Dart projects, but maybe since `deprecation.dart` is the only file like that here (other than the protobuf files, which use their own extension) just ignoring it explicitly is better.